testing/test_pythonpath.py

```diff
@@ -56,14 +56,16 @@ def test_two_dirs(pytester: Pytester, file_structure) -> None:
 
 
 def test_module_not_found(pytester: Pytester, file_structure) -> None:
-    pytester.makefile(".ini", pytest="[pytest]\n")  # no pythonpath listed
+    # if pythonpath setting not there, test should error
```

Comment:
lets put the descriptions into docstrings instead of comments

we don't make any use of it yet, but it might be nice if at a later point a reporting tool could use this

Reply:
Thanks. I was wondering what the preference was.